examples/Training/python/Advanced/Create_Image_advanced.py

```diff
@@ -7,14 +7,18 @@
 # Use is subject to license terms supplied in LICENSE.txt
 #
 
+import sys
+import os
+sys.path.append(os.path.join('..', 'python'))
```

Comment:
Two thoughts to prevent the Python path workaround assuming it is in place for importing `Parse_OMERO_Properties`:
- either use relative imports e.g. `from ..Parse_OMERO_Properties import...`
- or maybe migrate all the `Advanced` examples together with the rest of the other examples